### PR TITLE
Promote VB What's new

### LINKED ISSUE #6772: Need Visual Studio release details
Visual Basic 15.3 and Visual Basic 15.5 do not document the release of Visual Studio that they come out in.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 272fdffd-2201-990f-387b-d37467e976cf
* Version Independent ID: 06703ef5-e10a-3fdd-da13-0ce99903a5e6
* Content: [What's new for Visual Basic](https://docs.microsoft.com/en-us/dotnet/visual-basic/getting-started/whats-new#visual-basic-153)
* Content Source: [docs/visual-basic/getting-started/whats-new.md](https://github.com/dotnet/docs/blob/master/docs/visual-basic/getting-started/whats-new.md)
* Product: **dotnet-visualbasic**
* GitHub Login: @rpetrusha
* Microsoft Alias: **ronpet**

### PR DESCRIPTION
**Pull request by rpetrusha**
## Promoted VB What's new

This PR:

- Moves the VB What's new topic up one level in the TOC to make it more discoverable.
- Corrects the casing and formatting of some headers and list items.
- Notes which versions of Visual Basic were released with which versions of Visual Studio.

Fixes #6772 

//cc @KathleenDollard

### CODE REVIEW

**Comment by KathleenDollard** [comment]
👍

File: docs/visual-basic/getting-started/whats-new.md [hunk 1]
```diff
@@ -153,16 +153,16 @@ You can test for null in a very light syntactic way before performing a member a
 [Multi-line String Literals](../../visual-basic/programming-guide/language-features/strings/string-basics.md)  
  String literals can contain newline sequences.  You no longer need the old work around of using `<xml><![CDATA[...text with newlines...]]></xml>.Value`  
   
-Comments  
-You can put comments after implicit line continuations, inside initializer expressions, and amongst LINQ expression terms.  
+**Comments**  
+You can put comments after implicit line continuations, inside initializer expressions, and among LINQ expression terms.  
   
- Smarter Fully-qualified Name Resolution  
+**Smarter Fully-qualified Name Resolution**
```
Comment (JRAlexander) [inline docs/visual-basic/getting-started/whats-new.md]: Headings should be sentence-case.

File: docs/visual-basic/getting-started/whats-new.md [hunk 2]
```diff
@@ -153,16 +153,16 @@ You can test for null in a very light syntactic way before performing a member a
...
-You can put comments after implicit line continuations, inside initializer expressions, and amongst LINQ expression terms.  
+**Comments**  
+You can put comments after implicit line continuations, inside initializer expressions, and among LINQ expression terms.  
   
- Smarter Fully-qualified Name Resolution  
+**Smarter Fully-qualified Name Resolution**  
  Given code such as `Threading.Thread.Sleep(1000)`, Visual Basic used to look up the namespace "Threading", discover it was ambiguous between System.Threading and System.Windows.Threading, and then report an error.  Visual Basic now considers both possible namespaces together.  If you show the completion list, the Visual Studio editor lists members from both types in the completion list.  
   
- Year-first Date Literals  
+ **Year-first Date Literals**
```
Comment (JRAlexander) [inline docs/visual-basic/getting-started/whats-new.md]: Heading.

File: docs/visual-basic/getting-started/whats-new.md [hunk 3]
```diff
@@ -153,16 +153,16 @@ You can test for null in a very light syntactic way before performing a member a
...
- Smarter Fully-qualified Name Resolution  
+**Smarter Fully-qualified Name Resolution**  
  Given code such as `Threading.Thread.Sleep(1000)`, Visual Basic used to look up the namespace "Threading", discover it was ambiguous between System.Threading and System.Windows.Threading, and then report an error.  Visual Basic now considers both possible namespaces together.  If you show the completion list, the Visual Studio editor lists members from both types in the completion list.  
   
- Year-first Date Literals  
+ **Year-first Date Literals**  
  You can have date literals in yyyy-mm-dd format, `#2015-03-17 16:10 PM#`.  
   
- Readonly Interface Properties  
+ **Readonly Interface Properties**
```
Comment (JRAlexander) [inline docs/visual-basic/getting-started/whats-new.md]: Heading here.

File: docs/visual-basic/getting-started/whats-new.md [hunk 4]
```diff
@@ -171,7 +171,7 @@ You can put comments after implicit line continuations, inside initializer expre
  [#Disable Warning \<ID> and #Enable Warning \<ID>](../../visual-basic/language-reference/directives/directives.md)  
  You can disable and enable specific warnings for regions within a source file.  
   
- XML Doc-comment Improvements  
+ **XML Doc-comment Improvements**
```
Comment (JRAlexander) [inline docs/visual-basic/getting-started/whats-new.md]: Heading here.

File: docs/visual-basic/getting-started/whats-new.md [hunk 5]
```diff
@@ -183,10 +183,10 @@ You can put comments after implicit line continuations, inside initializer expre
  [Overrides Definitions are Implicitly Overloads](../../visual-basic/language-reference/modifiers/overrides.md)  
  If you add the `Overrides` modifier to a definition, the compiler implicitly adds `Overloads` so that you can type less code in common cases.  
   
- CObj Allowed in Attributes Arguments  
+ **CObj Allowed in Attributes Arguments**
```
Comment (JRAlexander) [inline docs/visual-basic/getting-started/whats-new.md]: Heading here

File: docs/visual-basic/getting-started/whats-new.md [hunk 6]
```diff
@@ -183,10 +183,10 @@ You can put comments after implicit line continuations, inside initializer expre
  [Overrides Definitions are Implicitly Overloads](../../visual-basic/language-reference/modifiers/overrides.md)  
  If you add the `Overrides` modifier to a definition, the compiler implicitly adds `Overloads` so that you can type less code in common cases.  
   
- CObj Allowed in Attributes Arguments  
+ **CObj Allowed in Attributes Arguments**  
  The compiler used to give an error that CObj(…) was not a constant when used in attribute constructions.  
   
- Declaring and Consuming Ambiguous Methods from Different Interfaces  
+ **Declaring and Consuming Ambiguous Methods from Different Interfaces**
```
Comment (JRAlexander) [inline docs/visual-basic/getting-started/whats-new.md]: Heading here.

**Review by JRAlexander — CHANGES_REQUESTED** [review]
Looks good, @rpetrusha, except for the headings. They should only have the first word capitalized.

**Comment by rpetrusha** [comment]
I've addressed your comments, @JRAlexander, if you'd like to take a look again.
@davkean, I've added Visual Studio versions for the last Visual Basic releases.
@KathleenDollard, you might want to take a look at the latest changes.

**Review by JRAlexander — APPROVED** [review]

**Comment by JRAlexander** [comment]
LGTM, @rpetrusha!

**Pull request merged**